.eslintrc.js

```diff
@@ -104,7 +104,7 @@ module.exports = {
     'jsdoc/check-tag-names': [
       'error',
       {
-        definedTags: ['plugin', 'util', 'experimental', 'deprecated', 'preserve', 'core', 'TODO', 'category']
+        definedTags: ['plugin', 'util', 'experimental', 'deprecated', 'preserve', 'core', 'TODO', 'category', 'internal']
```

Comment:
I'm not sure whether thus tag is needed,

Reply:
`@private` is for only in specific class usages. Using private members outside a class is incorrect.
`@internal` allows using members outside a class as long as are still inside a library.

What is that mean in a practical manner? I will show on the video:

https://user-images.githubusercontent.com/2893099/142199427-3f2cf491-b690-48b3-8d29-4d5c2dbcc558.mp4

Begin with `@internal`, inteliSense works.
Then change into `@private`, inteliSense stopped working.

Reply:
I found that the `@package` should be used for this reason.  It has the same meaning as `@internal`, and additionally, it is natively supported by jsdoc.

https://jsdoc.app/tags-package.html

Reply:
Fixed within f86f1e46cbb2a4e10ade2b8191f70402944aedaa